images/base/files/kind/bin/undo-mount-hacks.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# don't run on cgroups v2
+if [[ -f "/sys/fs/cgroup/cgroup.controllers" ]]; then
+    exit 0
+fi
+
+# on cgroups v1 we want to undo bind mounting over this to hide misc
+# see entrypoint
```

Comment:
this should probably `|| true` or otherwise check if we actually have a bind mount to remove, instead of only checking if we're on cgroups v1.

Reply:
done. went with `|| true`, because doing this is harmless if it's not a mount and not worth the complexity to detect.